app/src/main/java/org/luxoft/sdl_core/MainActivity.java

```diff
@@ -351,4 +308,35 @@ private void showToastMessage(String message) {
         toast.show();
     }
 
+    private final BroadcastReceiver mainActivityReceiver = new BroadcastReceiver() {
+        @Override
+        public void onReceive(Context context, Intent intent) {
+            String action = intent.getAction();
```

Comment:
```suggestion
            final String action = intent.getAction();
```

Reply:
fixed in f874c10935f6b28d6931e867ca35d2992f8b0469